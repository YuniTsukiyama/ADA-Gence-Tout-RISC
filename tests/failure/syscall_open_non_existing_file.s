main:
    mov %A, $3
    mov %B, file
    mov %C, $2
    syscall

    exit

.data
    file "non_existing_file.in"
