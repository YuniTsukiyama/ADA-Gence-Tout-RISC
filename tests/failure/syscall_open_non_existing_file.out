Error with the filename. Maybe it does not exists?
